repository/s.json

```
@@ -3938,6 +3938,17 @@
 				}
 			]
 		},
+		{
+			"name": "Sublime Swan",
+			"details": "https://github.com/efe-blue/sublime-swan",
+			"labels": ["swan", "miniapp", "highlight", "autocomplete", "baidu"],
+			"releases": [
+				{
+					"sublime_text": ">=3000",
+					"branch": "master"
```

```suggestion
					"tags": true
```

You may only use the tag-based release strategy. I see that you pushed a tag already, but make sure that you push another one with the recent changes.

---

@yufeng04 Please resolve this.

---

@Thom1729 I have pushed another tag with the recent changes

---

You must use `"tags": true` rather than `"branch": "master"`. Branch releases are no longer supported.

---

@Thom1729 Could you help me? I don't know the reason that all checks have failed!